provision-vm.yml

```diff
@@ -54,7 +53,7 @@
     - { role: welcome,           tags: ['welcome'] }
     - { role: openconext-common, tags: ['eb','sr'] }
     - { role: engineblock,       tags: ['eb'     ] }
-    - { role: vm_only_bootstrap_ldap_attributes,       tags: ['ldap'     ] }
+#    - { role: vm_only_bootstrap_ldap_attributes,       tags: ['ldap'     ] }
```

Comment:
Remnant of testing. It is duplicated two lines below and caused the 2nd to consistently fail (which also was ignored). Tried it without the first invocation (commented it out as it is currently), and it seems to keep on working correctly...

Perhaps @thijskh can shed some light on whether or not this is/could be an issue?